### ISSUE TITLE
I want to use YOLOv8 as base model to run inference using a custom trained model and save the results as a dataset.

### ISSUE THREAD

**Issue by AmmaraRazzaq** [autodistill/autodistill#89]
### Search before asking

- [X] I have searched the Autodistill [issues](https://github.com/autodistill/autodistill/issues) and found no similar feature requests.

### Description

right now YOLOv8 model is only being used as a target model. I want to use it as a base model so that I can load a pretrained model and use it to label the dataset. 

### Use case

This will help me in generating new examples to finetune the model.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by capjamesg** [comment]
Hello there! We don't currently support using YOLOv8 as a base model. This is not on our immediate roadmap, but a few people have suggested it so we will revisit the idea! We are prioritizing multimodal models as new ones come out that have utility in model distillation.

With that said, we do support using YOLOv8 models to label data using the [Autodistill Roboflow Universe](https://github.com/autodistill/autodistill-roboflow-universe) extension. You can upload your model weights to Roboflow, then use the hosted API we offer to run inference.

**Comment by capjamesg** [comment]
This functionality is now supported. Upgrade your Autodistill YOLOv8 package using `pip install --upgrade autodistill-yolov8`. Then, use the following code to auto-label data with a YOLOv8 model:

```python
from autodistill_yolov8 import YOLOv8Base
from autodistill.detection import CaptionOntology

# define an ontology to map class names to our YOLOv8 classes
# the ontology dictionary has the format {caption: class}
# where caption is the prompt sent to the base model, and class is the label that will
# be saved for that caption in the generated annotations
# then, load the model

# replace weights_path with the path to your YOLOv8 weights file
base_model = YOLOv8Base(ontology=CaptionOntology({"car": "car"}), weights_path="yolov5s.pt")

# run inference on a single image
results = base_model.predict("mercedes.jpeg")

base_model.label(
  input_folder="./images",
  output_folder="./dataset"
)
```

Let me know if you have any questions.
